docs/API.rst

```diff
@@ -39,6 +40,9 @@ Client
     If you set `selfSigned` to true SSL accepts certificates from a non trusted CA.
     If you set `certExpired` to true, the bot connects even if the ssl cert has expired.
 
+    `localAddress` is the address to bind to when connecting. Not
```

Comment:
It looks like you got this working with the secure option. Probably should update this doc.